src/webob/acceptparse.py

```diff
@@ -255,11 +294,382 @@ class AcceptEncoding(Accept):
     """
 
 
-class AcceptLanguage(Accept):
+class AcceptLanguage(object):
     """
-    Represents an ``Accept-Language`` header.
+    Represent an ``Accept-Language`` header.
+
+    Base class for :class:`AcceptLanguageValidHeader`,
+    :class:`AcceptLanguageNoHeader`, and :class:`AcceptLanguageInvalidHeader`.
     """
-    def _match(self, mask, item):
+
+    @classmethod
+    def python_value_to_header_str(cls, value):
```

Comment:
It feels like maybe method and `parse` are a little ambiguous. It's semi-feeling like there should just be one api that takes a python value and returns an accept language header. From there you can use `.header_value` to get the actual header string.

```python
header = AcceptLanguage.parse(...) # possibly more clear than the current `create_accept_language_header`
items = AcceptLanguage.parse(...).items # basically the same as `.parsed` right now
header_value = AcceptLanguage.parse(...).value # equivalent to `python_value_to_header_str`
```